python/ray/tune/examples/NASCifar_Trainable.py

```diff
@@ -0,0 +1,100 @@
+""" Trainable using NASCifar Benchmark from Tabular Benchmarks for Hyperparameter Optimization and Neural Architecture Search """
+from __future__ import print_function
+
+import os
+from ray.tune import Trainable
+from NASCifar10 import NASCifar10A, NASCifar10B, NASCifar10C
+import urllib.request
+
+
+class NASCifar10ATrainable(Trainable):
```

Comment:
For all of these, can you have the same base class and just have 3 classes that inherit from that with different data URL class

also for this base class can you use `_setup` instead of __init__?

Then